app/src/main/res/values/strings.xml

```diff
@@ -91,11 +91,11 @@
     <string name="fuck_off_button_content_description">This is the famous fuck off button.</string>
     <string name="fuck_off_dialog_title">The famous fuck off button</string>
     <string name="fuck_off_dialog_text">
-        Warning! This action cannot be undone! You will never ever hear or see this person again.\n
+        Warning! This action cannot be undone! You will never hear from this person again.\n
```

Comment:
I see what you are doing from a language perspective. An English teacher would correct "never ever under any circumstances even when hell freezes over" as redundant and replace it with "no." Sometimes, like when telling someone not to press the nuke launch button, a simple "no" doesn't quite do.

You can probably re-write the whole dialog message to deliver the message more effectively. So here is the goal for the message: Really discourage users from pressing the "fuck off button."

The reasons are this:
- A casual user may think that this works like on another dating app where the user gets more or less equivalent options. There discarding a match means that potentially something better pops up. Beeline doesn't work like that.

- In the regular course of actions, the comparisons of the user should already sort out whom the user is interested in. So the matches that they see really are matches that the users decided are their favorites.

- If the users are unhappy about these "favorites" because they didn't get liked back by their first choices, dismissing these choices will only get them worse choices. So that will set up a bad user experience. "Ugh, I hate Joe. Let me dismiss him. Oh no, Roger is even worse. Go away! Oh my god, what's wrong with Leo. Why do the choices keep getting worse and worse???" Well you already ranked your choices and your dismissing the ones on the top of your list!

- Also, the user should realize that they cannot undo this action ever. So tread carefully. We don't want people to pick around back and forth. We want users to give the top 10 matches their full attention and energy. (That's how we avoid the hundreds of messages going around with no or little response.)

- Ideally, this button only gets used after the users message each other and find out that the don't like each other. "Well, his profile was really exciting. But the BO on the date made even the flowers on the table wilt. I pass."


It's not necessary to explain all of this. The key objective is that the user is very hesitant to use the button, kind of understands the ranking impact, and is completely fine to dismiss users after they messaged/dated. Also, the encouragement should be to really focus the energy on the ten matches because they have the highest chance of going somewhere.